doc/source/user_guide/make_container.rst

```diff
@@ -0,0 +1,117 @@
+
+
+Procedure
+=========
+
+Specify the Ansys installation directory
+----------------------------------------
+
+Specify the pre-installed Ansys directory in the following function of 
+`copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_. 
+
+.. code:: python
+
+    copy_files(src="<path to ``ansys_inc`` directory>")
+
+Copy needed files
+-----------------
+
+Change local directory where `copy_docker_files.py <https://github.com/ansys/pyfluent/blob/main/docker/fluent/copy_docker_files.py>`_ 
+is located and run this script to copy needed files from the Ansys installation directory 
+to the container.
+
+.. code:: python
+
+    python copy_docker_files.py
+
+Not all the installation files are copied, the files ignored during the copying are 
+detailed in the following files.
+
+  * `excludeCEIList.txt <https://github.com/ansys/pyfluent/blob/main/docker/fluent/excludeCEIList.txt>`_
+  * `excludeFluentList.txt <https://github.com/ansys/pyfluent/blob/main/docker/fluent/excludeFluentList.txt>`_
+
+Build Docker image
+------------------
+
+Execute the following command where `Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_ is 
+located to build ``ansys_inc`` Docker image in the current directory.
+
+.. code:: console
+
+    sudo docker build -t ansys_inc:v251 .
+
+The Docker container configuration needed to build the container is detailed in the
+`Dockerfile <https://github.com/ansys/pyfluent/blob/main/docker/fluent/Dockerfile>`_.
+
+
+Run Docker container using the command line
+-------------------------------------------
+
+Please note, that you will have to specify the Ansys license file.
+
+Execute the following command to run the Docker container in solver mode.
+
+.. code:: console
+
+    sudo docker run -it --name ansys-inc -e ANSYSLMD_LICENSE_FILE=<license>.ansys.com ansys_inc:v251 3ddp -gu
+
+Execute the following command to run the Docker container in meshing mode.
+
+.. code:: console
+
+    sudo docker run -it --name ansys-inc -e ANSYSLMD_LICENSE_FILE=<license>.ansys.com ansys_inc:v251 3ddp -gu -meshing
+
+
+Run Docker container using PyFluent
+-----------------------------------
+
+Install `PyFluent <https://github.com/ansys/pyfluent>`_ and execute the following code
+to run the Docker container using PyFluent.
+
+.. code:: python
+
+    import os
+    import ansys.fluent.core as pyfluent
+    os.environ["ANSYSLMD_LICENSE_FILE"] = "<license>.ansys.com"
+    custom_config = {'fluent_image': 'ansys_inc:v251', 'host_mount_path': f"{os.getcwd()}", 'auto_remove': False}
+    solver = pyfluent.launch_fluent(container_dict=custom_config)
+
+
+Summary
+=======
+
+
+* **Step 1:** Specify Ansys installation directory.
+
+* **Step 2:** Copy needed files.
+
+* **Step 3:** Build Docker image.
+
+* **Step 4:** Run the Docker container using the command line or run the Docker container using PyFluent.
```

Comment:
In procedural information, you tell the user what to do in steps a-z. No summary is needed. By the time readers get to the end of the procedure, they've done all this.

Reply:
Removed. Thank you.